pkg/agent/flags/keys/keys.go

```diff
@@ -45,6 +45,7 @@ const (
 	UpdateDirectory                FlagKey = "update_directory"
 	ExportTraces                   FlagKey = "export_traces"
 	TraceSamplingRate              FlagKey = "trace_sampling_rate"
+	TraceBatchTimeout              FlagKey = "trace_batch_timeout"
```

Comment:
Do we need this as an exported flag?

Reply:
You're right -- removed from pkg/launcher/options.go, which will keep it from being a command-line/config flag. (We still need it here for all the control server setting stuff to work as expected -- this is the key it's stored under for the control server.)

Reply:
I feel like we don't have great idioms yet.